.gitignore

```diff
@@ -12,6 +12,7 @@ core/App/enclave_hsm_u.*
 core/Enclave/enclave_hsm_t.*
 core/App/*.o
 core/Enclave/*.o
+core/Test/*.o
 
 utils/tkey_exchange/*.o
```

Comment:
commit title: 
“
Split the unit test into function test and performance test

test: passed the unit test
Signed-off-by: yidingluo1 <yidingx.luo@intel.com>
”